blueprints/showbie-tooling/files/.template-lintrc.js

```diff
@@ -1,7 +1,24 @@
 module.exports = {
-  extends: 'recommended',
-  ignore: [],
+  // extends: 'recommended',
```

Comment:
We should turn on the `recommended` rules by default. If consuming applications need to disable it for whatever reason they can override this.